lib/mutant.rb

```diff
@@ -200,7 +200,14 @@ class Config
       fail_fast:         false,
       includes:          EMPTY_ARRAY,
       integration:       Integration::Null,
-      isolation:         Mutant::Isolation::Fork,
+      isolation:         Mutant::Isolation::Fork.new(
+        devnull: ->(&block) { File.open(File::NULL, File::WRONLY, &block) },
```

Comment:
As we later do `$stderr.repoen(devnull)` the flags have to be match up with the state of `$stderr` so we have to open with `w`.

```
/home/mbj/devel/mutant/lib/mutant/isolation.rb:68:in `reopen': <STDERR> can't change access mode from "w" to "a" (ArgumentError)
```